docs/layouts/shortcodes/deprecated-versions.md

```diff
@@ -0,0 +1,4 @@
+### Deprecated versions
+
+The following versions are available but not receiving security updates from upstream.
+You shouldn't use them and they will be removed at some point in the future.
```

Comment:
Thanks so much for adding a section for deprecated versions!
Shall we add something about people using the list of supported versions or migrating to those soon?